communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "ballista.png",
```

Comment:
After merging, your project will be featured on our [Discord ❤ Open Source](https://discord.com/open-source) page. It automatically applies a filter to your logo that makes it solid white except for the transparent portions.

The image you've included will look incorrect when this filter is applied. Could you swap the image out for a version with a transparent background that will look correct when solid white on a dark background?